packages/wrangler/src/init.ts

```diff
@@ -529,8 +586,15 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 					recursive: true,
 				});
 
+				const serviceMetaData = await fetchResult<ServiceMetaDataRes>(
+					`/accounts/${accountId}/workers/services/${fromDashScriptName}`
+				);
+				const defaultEnvironment =
+					serviceMetaData["default_environment"]["environment"];
+
+				// I want the default environment, assuming it's the most up to date code.
 				const dashScript = await fetchDashboardScript(
-					`/accounts/${accountId}/workers/scripts/${fromDashScriptName}`,
+					`/accounts/${accountId}/workers/services/${fromDashScriptName}/environments/${defaultEnvironment}/content`,
```

Comment:
I'm sure I'm missing some context here, but why is this section duplicated from above (starting line 513)? Could it be extracted into a function or something?

Reply:
Potentially a good future refactor.